lib/plug/error_handler.ex

```diff
@@ -94,7 +94,8 @@ defmodule Plug.ErrorHandler do
         |> handle_errors.(%{kind: kind, reason: normalized_reason, stack: stack})
     end
 
-    :erlang.raise(kind, reason, stack)
+    normalized_reason = Exception.normalize(kind, reason, stack)
```

Comment:
It would be best if we didn't  do it twice. Maybe we can move this to the topic and remove the normalization in the `after` clause?